tests/integration/API/PostFile.test.ts

```diff
@@ -0,0 +1,63 @@
+/* tslint:disable:no-relative-imports */
```

Comment:
This test file is not really testing the API/PostFile, is it?

Reply:
Not sure what you mean, this matches the other files in the folder

Reply:
Nevermind, my bad